docs-user/guide-remote-profiling.md

```diff
@@ -24,3 +24,5 @@ Make any necessary adjustments in the presented options, like threads to sample
 # Tips
 
 * Avoid opening any other panels besides the Performance panel, to reduce the overhead of activated panels.
+* After connecting to the device, do not select *Main Process* or any of the open tabs from the list of targets on the left.
+* If WebIDE automatically reconnects to a debugging target from a previous session, disconnect using the wrench icon on the top toolbar before clicking on *Performance*.
```

Comment:
If I understood everything properly, the wrench icon doesn't disconnect but close the toolbox. It _should_ remove the various actors from the target firefox, but we're not completetly sure of this, and I think that because we already tampered with the target Firefox, the profile session might be altered.

My advice is that if this happens the user should instead uncheck the 2 checkboxes in WebIDE's preferences and reconnect.

Reply:
Thanks for the feedback! This sounds concerning if it is indeed the case. When I was working on the toolbox/target code, destroying the toolbox was supposed to tear down everything on the server and release any resources held. So the only effect that could be observed afterwards would be potential increased memory pressure if the reconnection happened before the GC had a chance to kick in. Sounds like a bug we should fix if that is not the case.

But I do agree that your advice is reasonable and since WebIDE won't be around for much longer, I've updated the tip to that effect.